src/app/core/components/media-table/media-table.component.html

```diff
@@ -0,0 +1,106 @@
+<div class="table" role="table" aria-label="Media table">
```

Comment:
You should use the table html markup. comes with tons of semantic defaults to make it accessible. Something specific that dragged you into the div world her? Otherwise I would suggest to use tables, they are made for this specific use case.